src/components/with-concept/index.tsx

```diff
@@ -9,13 +9,15 @@ import type { Concept } from '../../types';
 export interface Props {
   concept: Concept | null;
   conceptActions: typeof actions;
+  isLoading: boolean;
 }
 
 const withConcept = (Component: ComponentType<any>) => {
   const WrappedComponent = (props: Props) => <Component {...props} />;
 
   const mapStateToProps = (state: any) => ({
-    concept: state.ConceptReducer.get('concept')?.toJS() ?? null
+    concept: state.ConceptReducer.get('concept')?.toJS() ?? null,
+    isLoading: state.ConceptReducer.get('isLoading')
```

Comment:
isLoading må namespaces for å unngå kollisjon med andre data providere. isLoading => isLoadingConcept